led_matrix_controller/content/audio_visualiser.py

```
@@ -0,0 +1,262 @@
+
+from __future__ import annotations
+
+import atexit
+import re
+from dataclasses import dataclass, field
+from json import dumps
+from multiprocessing import shared_memory
+from typing import Annotated, Final, Generator
+
+import numpy as np
+from content.setting import ParameterSetting, TransitionableParameterSetting
```

Ruff (0.7.0)

Move third-party import `content.setting.ParameterSetting` into a type-checking block

Move into type-checking block

(TCH002)